SemVer.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Created by Anita on 07/08/2016.
+ */
+
+myModuleI = {
+    SemVer: function (major, minor, revision) {
+        if (major < 0 || minor < 0 || revision < 0) {
+            throw new Error('Error: negative number' );
+        }else {
+            this.major = major;
```

Comment:
Declare variables in the beginning of the method. Here you're creating members inside and "else".